Fix stale test mocks and API mismatches

## Overview

Several test files have stale mocks that reference deleted components or outdated API signatures. This causes 4 test suites to fail.

## Failing Tests

### 1. Missing `OutputPanel` mock path

The following tests mock a component path that no longer exists:

- `src/__tests__/Smoke.test.tsx`
- `src/__tests__/MultiSelect.test.tsx`
- `src/__tests__/Interaction.test.tsx`

**Error:**
```
Cannot find module '../components/Panels/OutputPanel' from 'src/__tests__/XYZ.test.tsx'
```

**Fix:** Update or remove the mock for `OutputPanel` to match current component structure.

### 2. Playback API mismatch

`src/__tests__/handlePlayback.test.ts` references `pausePlayback()` which no longer exists.

**Error:**
```
TypeError: playback.pausePlayback is not a function
```

**Fix:** Update test mock to use current playback API (likely `stopPlayback()` or similar).

## Checklist

- [ ] Update/remove `OutputPanel` mocks in 3 test files
- [ ] Update `handlePlayback.test.ts` to match current playback API
- [ ] Verify all 320 tests pass

--- a/src/__tests__/Interaction.test.tsx
+++ b/src/__tests__/Interaction.test.tsx
@@ -24,10 +24,6 @@ jest.mock('../components/Toolbar/Toolbar', () => {
   });
 });
 
-jest.mock('../components/Panels/OutputPanel', () => () => (
-  <div data-testid="output-panel">Mock Output Panel</div>
-));
-
 // Mock Tone.js
 jest.mock('../engines/toneEngine', () => ({
   playNote: jest.fn(),

--- a/src/__tests__/MultiSelect.test.tsx
+++ b/src/__tests__/MultiSelect.test.tsx
@@ -9,7 +9,6 @@ import { createDefaultScore } from '@/types';
 jest.mock('../components/Toolbar/Toolbar', () => (props: any) => (
   <div data-testid="score-toolbar" />
 ));
-jest.mock('../components/Panels/OutputPanel', () => () => <div data-testid="output-panel" />);
 jest.mock('../hooks/usePlayback', () => ({
   usePlayback: () => ({
     isPlaying: false,

--- a/src/__tests__/Smoke.test.tsx
+++ b/src/__tests__/Smoke.test.tsx
@@ -22,13 +22,6 @@ jest.mock('../components/Toolbar/Toolbar', () => {
   });
 });
 
-// Mock OutputPanel
-jest.mock('../components/Panels/OutputPanel', () => {
-  return function MockOutputPanel() {
-    return <div data-testid="output-panel">Mock Output Panel</div>;
-  };
-});
-
 // Mock hooks
 jest.mock('../hooks/usePlayback', () => ({
   usePlayback: () => ({

--- a/src/__tests__/handlePlayback.test.ts
+++ b/src/__tests__/handlePlayback.test.ts
@@ -9,8 +9,10 @@ describe('handlePlayback', () => {
     mockPlayback = {
       playScore: jest.fn(),
       stopPlayback: jest.fn(),
+      pausePlayback: jest.fn(),
       isPlaying: false,
       lastPlayStart: { measureIndex: 0, eventIndex: 0 },
+      playbackPosition: { measureIndex: null, quant: null },
     };
     mockScore = {
       staves: [
@@ -55,7 +57,7 @@ describe('handlePlayback', () => {
     // Stop playback
     mockPlayback.isPlaying = true;
     handlePlayback(mockEvent, mockPlayback, selection, mockScore);
-    expect(mockPlayback.stopPlayback).toHaveBeenCalled();
+    expect(mockPlayback.pausePlayback).toHaveBeenCalled();
   });
 
   test('should ignore other keys', () => {

